### DISCUSSION TITLE
The "copy raw file" Feature Not Working

### DISCUSSION THREAD

**Discussion by Kajol-Kumari** [community/community#70128] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The `copy raw file` functionality doesn't work. On hover it does show the `copy raw file` but after click, nothing happens, ideally it should have copied the files content in clipboard
<img width="885" alt="Screenshot 2023-10-15 at 10 47 44 AM" src="https://github.com/community/community/assets/44888949/2d3a69d2-5742-40f2-a61b-2b8b58055de7">

**Comment by look** [comment]
Sorry about that. Can you provide some details about what browser and OS you're using? Thanks.

**Reply by Kajol-Kumari** [reply]
I am using google chrome and mac OS

**Comment by ghost** [comment]
To me, it is an issue from my isp (jio)
context: https://github.com/orgs/community/discussions/42655

I used cloudfare dns in browser settings and now copy works.

**Comment by astrobc1** [comment]
This feature is also broken for me on macos 14.6.1, Arc browser Version 1.61.0 (53949), Chromium Engine Version 129.0.6668.59. I also tried the latest version of firefox, no luck.
